### PR TITLE
Rework of the tests and other small changes

### PR DESCRIPTION
**Pull request by FeGeyer**
Here's the rework of our tests using Github actions instead of Travis now. After approval, we will delete the old test folder and rename the new one. Additionally, we addressed these issues:

- Deleted unnecessary save checkpoints in `model.py`
- Made use of the GPU keyword for the evaluation and checked if cuda is available
- Activate aborting of Training, if one does not want to overwrite an old model
- Deleted unnecessary `out_path` in `crop_first_component`
- Renamed area functions to ratio, as this is that is actually computed
- Corrected some typos
- Wrote new `create_predictions` function, which calls `get_predictions` or `get_seperate_predictions`, respectively
- Created `save_pred` and `read_pred`, because loading the model is slow
- Added a version number for `fastai` due to the removal of the `CudaCallback`
- Added pytest ordering

### CODE REVIEW

File: new_tests/test_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+        grid_x, grid_y = torch.meshgrid(a, a)
+        x_coords = torch.cat(num * [grid_x.flatten().unsqueeze(0)])
+        y_coords = torch.cat(num * [grid_y.flatten().unsqueeze(0)])
+        value = image.reshape(-1, pix ** 2)
+
+        assert x_coords.shape == (2, 3969)
+        assert y_coords.shape == (2, 3969)
+        assert value.shape == (2, 3969)
+
+    def test_bmul(self):
```
Comment (Kevin2) [inline new_tests/test_evaluation.py:140]: import bmul

File: new_tests/test_evaluation.py [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+        runner = CliRunner()
+        result = runner.invoke(main, "new_tests/evaluate.toml")
+        assert result.exit_code == 0
+
+        if os.path.exists("new_tests/model/evaluation"):
+            shutil.rmtree("new_tests/model/evaluation")
+
+
+def generate_images(conf):
```
Comment (Kevin2) [inline new_tests/test_evaluation.py:316]: nice

File: setup.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
     license="MIT",
     packages=find_packages(),
     install_requires=[
-        "fastai",
+        "fastai <= 2.3.0",
```
Comment (Kevin2) [inline setup.py:13]: bump version number

**Review by Kevin2 — APPROVED** [review]

File: new_tests/test_evaluation.py (hunk 1 above)
Comment (FeGeyer) [reply new_tests/test_evaluation.py:130]: Should be fixed in the new commits

File: setup.py (hunk 3 above)
Comment (FeGeyer) [reply setup.py:13]: Addressed in the latest commit

**Review by Kevin2 — APPROVED** [review]

**Pull request merged**